content/culture.md

```diff
@@ -1,8 +1,34 @@
 # Culture
 
-## implementation methods that affect diversity and inclusion
+High level description of corporate culture what it means and its importance.
 
-Methodologies
+focus on the primarily used corporate and organizational culture
+- Heiarchical
+  - position/senority based
+  - chain of command
+  - distance between workers and managers
+  - merit based reward system
+
+How corporate culture affects social culture.
+
+What is valued?
+  - employees
+  - ideas/innovation
+  - work output
+How are issues resolved/handled?
+  - isolated and quiet
+  - free to broadcast concerns
+  - protected from retaliation
+  - are they tracked to identify large scale problems
+  - just enough to be compliant and above board legally
+
+## Management communication approaches that affect diversity and inclusion
+
+Open door policies
+
+Tell HR
+
+Up to supervisor or manager to handle
```

Comment:
We probably need to noodle on some talking points for supervisors and managers at some point. They are likely going to be a first line of defense.

Reply:
Agreed and noted